src/FraudSessionInterface.php

```diff
@@ -0,0 +1,44 @@
+<?php
+
+namespace Drupal\commerce_bluesnap;
+
+/**
+ * Interface to process payments.
+ */
+interface FraudSessionInterface {
+
+  /**
+   * Returns the fraud session ID.
+   *
+   * A new ID will be generated if none exists yet.
+   *
+   * @return string
+   *   Bluesnap fraud session ID
+   */
```

Comment:
**CHANGE REQUEST**

When there is an interface the class docblock should simply have `{inheritdoc}`.

Reply:
Corrected